### DISCUSSION TITLE
Backend todos

### DISCUSSION THREAD

**Discussion by kylekim2123** [Java-and-Script/pickple-back#91] [General]
- [x] Enum 값에 대해 모든 Request DTO, Param에서 String이 아닌 Enum 객체로 입력
- [x] 양방향 연관관계 정의 및 일급 컬렉션 처리
- [ ] JPA Repository default 메서드 작성 및 서비스 코드 삭제
- [ ] AddressDepth1과 AddressDepth2를 하나로 합칠 수 있는 방법 구상
- [x] TODO 주석 완료
- [ ] Crew, Game 도메인에서 서로 비슷한 것들끼리 네이밍 형식 통일
- [ ] 각 API에 대한 인증, 인가 처리
- [x] CORS 이슈 처리
- [ ] 인가에 대한 예외 코드 따로 뺄것인지 논의

**Comment by kylekim2123** [comment]
@jay-so @charlesuu @Hchanghyeon

**Reply by Hchanghyeon** [reply]
확인했습니다.
